src/main/java/app/studnicki/ox/Main.java

```diff
@@ -0,0 +1,37 @@
+package app.studnicki.ox;
+
+class Main {
+  public static void main(String[] args) {
```

Comment:
Maybe some doc / comment on what it does now

Reply:
Added javadoc.